accessibility/house-style.md

```diff
@@ -72,13 +72,9 @@ As of January 18th 2018, WCAG 2.0 AA will be incorporated by reference into Sect
 
 You are expected to be aware of the Web Content Accessibility Guidelines and to adhere to their recommendations. You are also expected to understand the foundational four principles of accessibility (Perceivable, Operable, Understandable, Robust) and to be able to apply them in your work. Be alert to the possibility of regressions that cannot be detected by software.
 
-When implementing WCAG 2.0, we aim to comply with [WCAG 1](https://www.w3.org/TR/WCAG10/) AA and the [WCAG Samurai Errata](http://www.wcagsamurai.org/erratas/introduction/). This gives us a high pass rate with WCAG 2.0 AA. 
+The companion document [Understanding WCAG 2.0](https://www.w3.org/TR/UNDERSTANDING-WCAG20/Overview.html) explains what each of the success criterion in WCAG 2.0 actually means. You might also find the [Techniques for WCAG 2.0](https://www.w3.org/TR/WCAG20-TECHS/) useful when building features. You may find [WCAG 1](https://www.w3.org/TR/WCAG10/) AA plus the [WCAG Samurai Errata](http://www.wcagsamurai.org/erratas/introduction/) clearer and easier to understand. 
```

Comment:
It _used_ to be the case that WCAG 2.0 was hard to wrap your head around, but that "[Understanding WCAG 2.0](https://www.w3.org/TR/UNDERSTANDING-WCAG20/Overview.html)" document now does a really really good job of explaining it.

Feel free to remove the reference to WCAG 1.0 and the Samurai Errata.